src/transformers/utils/import_utils.py

```diff
@@ -35,6 +35,39 @@
 
 logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
 
+
+@overload
+def _is_package_available(pkg_name: str) -> bool:
+    ...
+
+
+@overload
+def _is_package_available(pkg_name: str, return_version: Literal[False]) -> bool:
+    ...
+
+
+@overload
+def _is_package_available(pkg_name: str, return_version: Literal[True]) -> Tuple[bool, str]:
+    ...
```

Comment:
We don't do this in Transformers. The full signature of the function is way more than enough for us (also this is a private method, so no one will look at its signature).